flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumer.java

```diff
@@ -41,7 +42,6 @@
 import org.apache.flink.streaming.connectors.kinesis.serialization.KinesisDeserializationSchema;
 import org.apache.flink.streaming.connectors.kinesis.serialization.KinesisDeserializationSchemaWrapper;
 import org.apache.flink.streaming.connectors.kinesis.util.KinesisConfigUtil;
-import org.apache.flink.streaming.util.serialization.DeserializationSchema;
```

Comment:
I think we can not simply remove the usage of this, as that will be an user API-breaking change (for users that use the `FlinkKinesisConsumer(String, DeserializationSchema<T>, Properties)` constructor).

Instead of removing this, we should deprecate the constructors that accept the deprecated `DeserializationSchema` interface, and add new corresponding constructors that accept the new migrated `DeserializationSchema`.

Reply:
make sense.